packages/react-test-app/index.html

```diff
@@ -7,15 +7,12 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
-    <link rel="apple-touch-icon" href="/logo192.png" />
-    <link rel="manifest" href="/manifest.json" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="theme-color" content="#000000" />
-    <title>React Test App</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>ReactTestApp</title>
+    <base href="/" />
   </head>
-  <body>
+  <body class="theme-classic-dark">
```

Comment:
Why is this class necessary?

Reply:
When the themeSwitcher package is utilized within the echarts example, the addition of this class becomes redundant.